app/views/application/_messages.html.erb

```diff
@@ -1,6 +1,3 @@
 <% if flash[:notice] %>
   <p class="flash-notice"><%= flash[:notice] %></p>
 <% end %>
-<% if flash[:error] %>
```

Comment:
The key `:error` isn't one of the standard categories, if we wanted to be future proof then you'd do something like this:

``` ruby
<% flash.each.key do |key, value| %>
  <%= content_tag(:p, value, class: "flash-#{key}" %>
<% end %>
```

But you know, YAGNI.